source-control/Bug_Reporting_TFS_2010.md

```diff
@@ -0,0 +1,520 @@
+
+**Recommendation**  
+The following recommendation targets all development streams and comes from a joint effort between developers and testers and is approved by the CS-03 Technical Advisors.
+
+**Scope of the recommendation:** This recommendation should be used by any person that performs testing activities against a solution and targets the default Agile Template of TFS 2010 for their bug reporting.  
+**Out of Scope:** Git technologies  
+**Definition:** Bug reporting is the activity of raising and tracking defects of an application or product.  
+**Usage:** Application development, Testing (clients, testing services, performance, security, web accessibility...), ALM  
+**Alternative use:**
+
+## Contents
+
+* [1 Recommendation](#Recommendation)
+  * [1.1 Assumptions](#Assumptions)
+  * [1.2 Tools](#Tools)
+  * [1.3 Actors](#Actors)
+    * [1.3.1 Reporter](#Reporter)
+    * [1.3.2 Dev Coordinator](#Dev_Coordinator)
+    * [1.3.3 Resolver](#Resolver)
+    * [1.3.4 TST Coordinator](#TST_Coordinator)
+    * [1.3.5 Validator](#Validator)
+  * [1.4 Fields of a Bug](#Fields_of_a_Bug)
+    * [1.4.1 Title:](#.2ATitle:)
+    * [1.4.2 Assign To:](#.2AAssign_To:)
+    * [1.4.3 States and Reasons](#.2AStates_and_.2AReasons)
+    * [1.4.4 Area](#.2AArea)
+    * [1.4.5 Iteration:](#.2AIteration:)
+    * [1.4.6 Classification:](#Classification:)
+    * [1.4.7 Stack Rank:](#Stack_Rank:)
+    * [1.4.8 Priority:](#Priority:)
+    * [1.4.9 Severity:](#Severity:)
+    * [1.4.10 System Info:](#.2ASystem_Info:)
+    * [1.4.11 Details:](#.2ADetails:)
+    * [1.4.12 Test Cases:](#Test_Cases:)
+    * [1.4.13 All Links:](#All_Links:)
+    * [1.4.14 Attachments:](#Attachments:)
+  * [1.5 Bug Reporting Process](#Bug_Reporting_Process)
+    * [1.5.1 Reporting the Bug](#Reporting_the_Bug)
+    * [1.5.2 Resolving the Bug](#Resolving_the_Bug)
+    * [1.5.3 Building and Deploying the new version](#Building_and_Deploying_the_new_version)
+    * [1.5.4 Validating the Resolution](#Validating_the_Resolution)
+  * [1.6 Bug Reporting Diagram](#Bug_Reporting_Diagram)
+  * [1.7 Team Queries](#Team_Queries)
+    * [1.7.1 Active Bugs](#Active_Bugs)
+    * [1.7.2 Bugs to be tested and/or closed](#Bugs_to_be_tested_and.2For_closed)
+  * [1.8 Build Definition Naming Convention (.Net Stack)](#Build_Definition_Naming_Convention_.28.Net_Stack.29)
+  * [1.9 Testing Complex or Multi-Application Solutions](#Testing_Complex_or_Multi-Application_Solutions)
+    * [1.9.1 Option 1: Provide access to your TFS](#Option_1:_Provide_access_to_your_TFS)
+    * [1.9.2 Option 2: Merge into 1 Team Project](#Option_2:_Merge_into_1_Team_Project)
+    * [1.9.3 Option 3: Create a Central Team Project](#Option_3:_Create_a_Central_Team_Project)
+  * [1.10 Template for "Details = Steps to Reproduce" Field](#Template_for_.22Details_.3D_Steps_to_Reproduce.22_Field)
```

Comment:
Not all these links work. I don't think we need all of them either; probably just the first level.